ui/app/vue.config.js

```diff
@@ -20,11 +20,8 @@ module.exports = {
     loaderOptions: {
       sass: {
         additionalData: `
-            @import "normalize-scss";
             @import "@/scss/typography.scss";
             @import "@/scss/variables.scss";
-            @import "@/scss/reset.scss";
-            @import "@/scss/utilities.scss";
```

Comment:
These were moved because they were writing out styles and everything here will be added to each individual component.